### PR TITLE
HF technical accuracy audit: Staking & Snarking

### PR DESCRIPTION
**Pull request by barriebyron**
A user reported the set-staking subcommand is in the docs, but does not show in mina client command line help. See my request for the protocol team: https://github.com/o1-labs/docs2/pull/539/files#diff-503718b7c9d2eebaea9795004b7bc33e73052f7fcc3e4162527ef340a6830536R44 

While I was reviewing the doc, I improved the content in preparation for ITN and HF. Please review for thoroughness and technical accuracy. Handy [Staking and Snarking](https://docs2-git-barriebyron-patch-1-minadocs.vercel.app/node-operators/staking-and-snarking)  preview link for your viewing pleasure

Related:
https://app.zenhub.com/workspaces/zkapps-product-eng-6130fedb3b0fc600123d8796/issues/gh/o1-labs/docs2/538

### CODE REVIEW

File: docs/node-operators/staking-and-snarking.mdx [hunk 1]
```diff
@@ -67,82 +93,90 @@ Addresses and ports:        ...
...
 
 :::
 
-If you'd like to send your coinbase to an account other than the one that is staking, you can use the `-coinbase-receiver` flag when you start your daemon. You can even point the coinbase at a cold wallet!
+If you want to send your coinbase to an account other than the one that is staking, use the `-coinbase-receiver` flag when you start your daemon. You can even point the coinbase at a cold wallet.
 
 ### Delegating MINA
 
-Delegating MINA is an alternative option to staking it directly, with the benefit of not having to maintain a node that is always connected to the network.
+Delegating MINA has the benefit of not having to maintain a node that is always connected to the network.
```
Comment (barriebyron) [inline docs/node-operators/staking-and-snarking.mdx:110]:
```suggestion
Delegating MINA has the benefit of not having to maintain a node that is always connected to the network. See [Mina Foundation Delegation Program](/node-operators/foundation-delegation-program).
```

File: docs/node-operators/staking-and-snarking.mdx [hunk 2]
```diff
@@ -1,45 +1,64 @@
...
+To synchronize your server software clock with internet time servers, be sure to set up NTP. Many popular Linux distributions include NTP as a default service.
 
-### Staking MINA
 
-We can try out staking with our MINA by issuing the following command:
+To stake with your MINA:
 
-    mina client set-staking --public-key $MINA_PUBLIC_KEY
+```sh
+mina client set-staking --public-key $MINA_PUBLIC_KEY
```
Comment (barriebyron) [inline docs/node-operators/staking-and-snarking.mdx:44]: protocol team, please verify that the set-staking subcommand is present in the command line help 
(or we can update the doc and instructions if the workflow is changed)

**Review by deepthiskumar — CHANGES_REQUESTED** [review]
`delegate-stake` is already part of the section for delegating stake. The set-staking section for staking one's own mina, aka, running block producer nodes. There was a command before to 
1. Enable staking on a node that did not have staking (using daemon command flags `--block-producer-key` or `--block-producer-pubkey`) 
or 
2. Change the public key with which validators wanted to stake 

We should simply remove the `set-staking` option and keep only the
`you can restart the daemon with the -block-producer-pubkey flag:` part as shown in the doc currently https://docs.minaprotocol.com/node-operators/staking-and-snarking

**Comment by barriebyron** [comment]
thanks @deepthiskumar  for that feedback, see changes for approval please

**Comment by barriebyron** [comment]
issue for the incorrect Mina Client CLI doc https://github.com/MinaProtocol/mina/issues/13820

File: docs/node-operators/staking-and-snarking.mdx [hunk 3]
```diff
@@ -1,45 +1,64 @@
...
+Network Time Protocol [(NTP)](https://en.wikipedia.org/wiki/Network_Time_Protocol) is the most common method for synchronizing computer clocks over a network. 
 
-### Staking MINA
+To synchronize your server software clock with internet time servers, be sure to set up NTP. Many popular Linux distributions include NTP as a default service.
 
-We can try out staking with our MINA by issuing the following command:
+## Set or change the block producer key
 
-    mina client set-staking --public-key $MINA_PUBLIC_KEY
+Use the `mina client delegate-stake` subcommand to set or change the block producer key. See the [Mina Client](/node-operators/mina-cli-reference#mina-client) CLI help.
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply docs/node-operators/staking-and-snarking.mdx]: great @georgeee  much clearer

File: docs/node-operators/staking-and-snarking.mdx [hunk 4]
```diff
@@ -1,45 +1,64 @@
...
+  - delegation
+  - delegate
+  - block rewards
 ---
 
-:::note
+# Staking and Snarking
 
-A new version of Mina Docs is coming soon! This page will be rewritten.
+You can interact with the Mina network by participating in consensus and helping compress data by generating zk-SNARKs. By operating a node that helps secure the network, you can receive MINA for your efforts. See [Staking Rewards on Mina](https://minaprotocol.com/blog/staking-rewards-on-mina).
```
Comment (georgeee) [inline docs/node-operators/staking-and-snarking.mdx:20]: I would propose to change "compress" to something more like "produce a cryptographic proof for the data".

Compression is only a very imprecise analogy.

**Review by georgeee — APPROVED** [review]

File: docs/node-operators/staking-and-snarking.mdx (hunk 4 above)
Comment (barriebyron) [inline docs/node-operators/staking-and-snarking.mdx:20]:
```suggestion
You can interact with the Mina network by participating in consensus to produce a cryptographic proof for the data by generating zk-SNARKs. By operating a node that helps secure the network, you can receive MINA for your efforts. See [Staking Rewards on Mina](https://minaprotocol.com/blog/staking-rewards-on-mina).
```

**Review by bkase — APPROVED** [review]
I did a quick skim, looks good!

**Comment by barriebyron** [comment]
@bkase thank you for approval... hmmm, still can't merge. can you help me unblock this PR?

> Merging is blocked
> Merging can be performed automatically once the requested changes are addressed.

File: docs/node-operators/staking-and-snarking.mdx [hunk 5]
```diff
@@ -1,39 +1,64 @@
...
+To properly remain synced to the network and participate in consensus, your server must run a [clock synchronization protocol](https://en.wikipedia.org/wiki/Clock_synchronization).
 
-Alternatively, you can restart the daemon with the `-block-producer-pubkey` flag:
+Network Time Protocol [(NTP)](https://en.wikipedia.org/wiki/Network_Time_Protocol) is the most common method for synchronizing computer clocks over a network. 
+
+To synchronize your server software clock with internet time servers, be sure to set up NTP. Many popular Linux distributions include NTP as a default service.
+
+## Set or change the block producer key
+
+Use the `mina client delegate-stake` subcommand to set or change delegation of the block producer key. See the [Mina Client](/node-operators/mina-cli-reference#mina-client) CLI help.
```
Comment (deepthiskumar) [inline docs/node-operators/staking-and-snarking.mdx:45]: I think it is better to point to the `Delegating MINA` that follows this section and which has more information about delegating

**Review by deepthiskumar — APPROVED** [review]
Made a suggestion but otherwise lgtm

File: docs/node-operators/staking-and-snarking.mdx (hunk 5 above)
Comment (barriebyron) [inline docs/node-operators/staking-and-snarking.mdx:45]:
```suggestion
Use the `mina client delegate-stake` subcommand to set or change the delegation of the block producer key. See
 [Delegating MINA](#delegating-mina).
```

**Pull request merged**